src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -36,14 +41,15 @@ protected function doClone($var)
         $maxItems = $this->maxItems;
         $maxString = $this->maxString;
         $cookie = (object) array();     // Unique object used to detect hard references
-        $gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
         $a = null;                      // Array cast for nested structures
         $stub = null;                   // Stub capturing the main properties of an original item value
                                         // or null if the original value is used directly
 
         if (!self::$hashMask) {
+            self::$gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
```

Comment:
If you want to, you can just read the current value with `(function () { var_dump(self::$gid); })->bindTo(null, VarCloner::class)();`, but I think I gave enough input, don't care to discuss it further.